### ISSUE TITLE
Mod_fifo pgsql

### ISSUE THREAD

**Issue by volga629** [signalwire/freeswitch#10]
Hello Everyone,
Latest release 1.10.1 mod_fifo can't connect to database pgsql. 
In log produce the follow messages.

That worked before upgrade.
```
<param name="odbc-dsn" value="$${dsn}"/>

```

```
2019-09-25 06:37:19.126246 [NOTICE] switch_event.c:2171 Event Binding deleted for mod_fifo:PRESENCE_PROBE
2019-09-25 06:37:19.126246 [NOTICE] switch_event.c:463 Subclass reservation deleted for mod_fifo.c:fifo::info
**2019-09-25 06:37:29.267854 [ERR] mod_fifo.c:4398 ODBC IS NOT AVAILABLE!**
2019-09-25 06:37:29.267854 [INFO] switch_core_sqldb.c:1870 fifo Starting SQL thread.
2019-09-25 06:37:29.267854 [ERR] switch_core_sqldb.c:702 [db="fifo",type="core_db"] NATIVE SQL ERR [no such table: fifo_outbound]
delete from fifo_outbound where static = 1 or taking_calls < 0 or stop_time < 0
2019-09-25 06:37:29.267854 [ERR] switch_core_sqldb.c:702 [db="fifo",type="core_db"] NATIVE SQL ERR [no such table: fifo_bridge]
delete from fifo_bridge
2019-09-25 06:37:29.267854 [ERR] switch_core_sqldb.c:702 [db="fifo",type="core_db"] NATIVE SQL ERR [no such table: fifo_callers]
delete from fifo_callers
2019-09-25 06:37:29.267854 [ERR] switch_core_sqldb.c:702 [db="fifo",type="core_db"] NATIVE SQL ERR [no such table: fifo_outbound]
update fifo_outbound set start_time=0,stop_time=0,ring_count=0,use_count=0,outbound_call_count=0,outbound_fail_count=0 where static=0
2019-09-25 06:37:29.267854 [ERR] switch_core_sqldb.c:702 [db="fifo",type="core_db"] NATIVE SQL ERR [no such table: fifo_outbound]

```

**Comment by andywolk** [comment]
That's already fixed in master: fd2f12cf94c4445843a299ef8b87ce2d15d77072
https://freeswitch.org/jira/browse/FS-11995

**Comment by volga629** [comment]
Is possible get diff to patch 1.10.1 ?
